From bf5692fc60cfacd17b2bc5e8f0c668c3b83ce8ff Mon Sep 17 00:00:00 2001
From: Jordan Petridis <jordan@centricular.com>
Date: Thu, 16 Dec 2021 22:43:27 +0200
Subject: [PATCH 2/2] meson: bump gettext package suffix

Currently, both gnome-bluetooth-1.0 and 2.0 are using "2"
as a suffix for the gettext locale files which causes them to
overalp. Looks like 2 has been used since the initial commit
of the meson port, back in 3.20 cycle, although not sure if
there is any special meaning to the number. Bump it anyway
so that translations won't overlap between the two versions.
---
 meson.build | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b7fd1e04..fddf173a 100644
--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,10 @@ gnomebt_api_name = '@0@-@1@'.format(meson.project_name(), gnomebt_api_version)
 
 gnomebt_gir_ns = 'GnomeBluetooth'
 
-gnomebt_gettext_package = meson.project_name() + '2'
+# The number at the end a suffix of the files and should
+# be bumped along the api_version so that translations
+# can be installed in parallel with older versions.
+gnomebt_gettext_package = meson.project_name() + '3'
 
 gnomebt_prefix = get_option('prefix')
 gnomebt_datadir = get_option('datadir')
-- 
2.31.1

